src/Compilers/CSharp/Portable/Emitter/Model/NamedTypeReference.cs

```diff
@@ -152,5 +152,19 @@ Cci.IDefinition Cci.IReference.AsDefinition(EmitContext context)
         {
             return null;
         }
+
+        CodeAnalysis.Symbols.ISymbolInternal Cci.IReference.GetInternalSymbol() => UnderlyingNamedType;
+
+        public sealed override bool Equals(object obj)
```

Comment:
We should probably also mark these as `Obsolete`, so that any direct usage is flagged at compile time.

Reply:
> We should probably also mark these as Obsolete, so that any direct usage is flagged at compile time.

We never operate on instances of this type, they are used as Cci entities.

---
In reply to: [512353520](https://github.com/dotnet/roslyn/pull/48838#discussion_r512353520) [](ancestors = 512353520)